### PR TITLE
Add documentation on how to schedule a flow using the AJAX API

### PR DESCRIPTION
**Pull request by erwa**
Here's what it looks like:
scheduleflow

### CODE REVIEW

File: _includes/docs/2.5/sidebar.html [hunk 1]
```diff
@@ -50,26 +50,27 @@
...
-                                        <li><a href="#api-execute-a-flow">Execute a Flow</a></li>
-                                        <li><a href="#api-cancel-a-flow-execution"> Cancel a Flow Execution</a></li>
-                                        <li><a href="#api-pause-a-flow-execution"> Pause a Flow Execution</a></li>
-                                        <li><a href="#api-resume-a-flow-execution"> Resume a Flow Execution</a></li>
-                                        <li><a href="#api-fetch-a-flow-execution"> Fetch a Flow Execution</a></li>
-                                        <li><a href="#api-fetch-execution-job-logs">Fetch Execution Job Logs</a></li>
-                                        <li><a href="#api-fetch-flow-execution-updates">Fetch Flow Execution Updates</a></li>
-                                    </ul>
-                            </li>
+              <li><a href="#ajax-api">AJAX API</a>
```
Comment (logiclord) [inline _includes/docs/2.5/sidebar.html]: Can you keep please indent <li> tag ?

**Comment by logiclord** [comment]
minor indentation.. LGTM

**Comment by erwa** [comment]
Fixed the indentation in `sidebar.html`. If you want to confirm I only added the api-schedule-a-flow link, I suggest installing the Chrome Octosplit plugin (https://chrome.google.com/webstore/detail/octosplit/mnkacicafjlllhcedhhphhpapmdgjfbb) and selecting "No whitespace". It's a really handy plugin for viewing diffs, especially when there are lots of whitespace changes.

**Comment by logiclord** [comment]
Cool plugin :+1:

LGTM.

**Comment by hluu** [comment]
Thanks for adding this Anthony.

**Comment by erwa** [comment]
A user was confused about how to find the projectId, so I clarified the documentation a bit in https://github.com/azkaban/azkaban/pull/392

**Pull request merged**